modules/@apostrophecms/ui/ui/apos/components/AposSlat.vue

```diff
@@ -18,11 +18,24 @@
     >
       <div class="apos-slat__main">
         <drag-icon class="apos-slat__control apos-slat__control--drag" :size="13" />
+        <a
+          class="apos-slat__control apos-slat__control--view"
+          v-if="item.url"
+          :href="item.url"
+        >
+          <eye-icon :size="14" />
+        </a>
+        <div v-if="item.ext" class="apos-slat__extension-wrapper">
+          <span class="apos-slat__extension" :class="[`apos-slat__extension--${item.ext}`]">
+            {{ item.ext }}
+          </span>
+        </div>
         <div class="apos-slat__label">
           {{ item.title }}
         </div>
       </div>
       <div class="apos-slat__secondary">
+        <div class="apos-slat__size" v-if="item.size">{{ item.size }}</div>
         <close-icon
```

Comment:
This close icon needs to be a button. You could use the `AposButton` and the icon props, or wrap this directly here.

Reply:
made a new `inline` button modifier that is basically an AposButton with no chrome